frontend/index_es.html

```diff
@@ -0,0 +1,551 @@
+          const param_study_name = new URL(connectUrl).searchParams.get("study_config");
+          return param_study_name == null? "stage-program" : param_study_name;
+        }
+        const orig_host = new URL(connectUrl).hostname;
+        // If we are running this on dev, use a param instead to enable testing
+        if (orig_host == "localhost") { return _getFromParamOrDefault(); }
+        const first_domain = orig_host.split(".")[0];
+        if (first_domain == "openpath-stage") { return _getFromParamOrDefault(); }
+        const openpath_index = first_domain.search("-openpath");
+        if (openpath_index == -1) { return undefined; }
+        const study_name = first_domain.substr(0,openpath_index);
+        return study_name;
+      }
+			$(function() {
+				$(document).ready(function() {
+					const STUDY_NAME = _getStudyName(window.location);
+          const JOIN_CODE = `nrelopenpath://join_study?label=${STUDY_NAME}&source=github`
+
+          // Generate study-join QR code
+          var qrcode = new QRCode(document.getElementById("qrcode"), {
+	          text: JOIN_CODE,
+	          width: 100,
+	          height: 100,
+	          colorDark : "#000000",
+	          colorLight : "#ffffff",
+	          correctLevel : QRCode.CorrectLevel.H
+          });
+          $.getJSON(`https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/${STUDY_NAME}.nrel-op.json`, function(data) {
+            
+            // Title card
+            const PAGE_TITLE_TEXT = `${data.intro.translated_text.es.deployment_name} Pagina Inicial` // The sentence "Pagina Inicial" was hardcoded do lack of token translation in .json file. This will be discussed with Shankari.
+            $('#pageTitle').text(PAGE_TITLE_TEXT)
+            document.title = PAGE_TITLE_TEXT;
+            
+            // Navigation links
+            const PUBLIC_DASHBOARD_LINK = data.server.connectUrl.replace("api", "public");;
+            $('#publicDashboardLink').attr('href', PUBLIC_DASHBOARD_LINK);
+
+            // Deployment partner content text
+            const DEPLOYMENT_PARTNER_NAME = `${data.intro.deployment_partner_name}`;
+            $('#deploymentPartnerName').text(DEPLOYMENT_PARTNER_NAME);
+            
+            // Program or study specific differences
+            const PROGRAM_OR_STUDY = `${data.intro.program_or_study}`;
+            if(PROGRAM_OR_STUDY == 'study') {
+              $('#tokenType').text("an autogenerated ");
+            } else if(PROGRAM_OR_STUDY == 'program') {
+              $('#tokenType').text("a program-provided ");
+            }
+            $('#programOrStudy').text(PROGRAM_OR_STUDY);
+
+            // Study title content text
+            const STUDY_TITLE_TEXT = DEPLOYMENT_PARTNER_NAME + ` ${data.intro.translated_text.es.deployment_name}`;
+            $('#studyTitleOne').text(STUDY_TITLE_TEXT);
+            $('#studyTitleTwo').text(STUDY_TITLE_TEXT);
+
```

Comment:
I see that you are using the `data.intro.translated_text.es.deployment_name` which is correct. But there is no `studyTitleOne` to fill it into.

Reply:
Yes, we didn't use #studyTitleOne, but as mentioned above, we will undo this change.